php/boot-phar.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+// Store the path to the Phar early on for `Utils\phar-safe-path()` function.
+define( 'WP_CLI_PHAR_PATH', getcwd() );
```

Comment:
> This gives us the bit in front of the phar name, and it even works with different phar names.

I know. I'm just afraid of `getcwd()` sometimes producing a different value than we expect.